workflow/controller/operator.go

```diff
@@ -312,6 +312,12 @@ func (woc *wfOperationCtx) operate(ctx context.Context) {
 			woc.failSuspendedAndPendingNodesAfterDeadlineOrShutdown()
 		}
 
+		if err != nil && strings.Contains(err.Error(), "need reconcil workflowtaskresults") {
+			woc.log.WithError(err).WithField("workflow", woc.wf.ObjectMeta.Name).Error("need reconcil workflowtaskresults")
+			woc.requeue()
+			return
+		}
```

Comment:
I don't think we should ever rely on the specific wording of an error - if needed, we could have specific error types that can be checked or something

Reply:
also, I see you're logging an error - but it's not actually an error, right?

Reply:
just wanted to make sure you saw the second comment above as well - if it's not an error, let's just log it as a debug line

Reply:
ok,I don't need a error. But I want to make it a info. This information is not common. I can search for this information to see if my cluster is facing this "need reconcile workflowtaskresults" problem. Now I do the same，help me determine the pressure on the cluster at this time.
![image](https://github.com/argoproj/argo-workflows/assets/72060326/29190446-91f2-4d8e-9807-5ac6eba313c2)

Reply:
okay, I'm good with "Info" level